code/src/componenets/AddTask.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import tasks from 'reducers/task';
+import styled from 'styled-components/macro';
+import TaskList from './TaskList';
+
+const AddTask = () => {
+  const [inputValue, setInputValue] = useState('');
+  const dispatch = useDispatch();
+  const onFormSubmit = (event) => {
+    event.preventDefault();
+    const newTask = { id: Date.now().toString(),
```

Comment:
The date is not showing on the page, so you could either remove this or print it out (I never managed to print it out....)